lib/collection/src/collection_manager/optimizers/segment_optimizer.rs

```diff
@@ -1,3 +1,4 @@
+use std::borrow::Borrow;
```

Comment:
this doesn't look related, is it?

Reply:
Yes, I think it is added by mistake. I will remove it.